src/Raven.Server/Documents/BlittableMetadataModifier.cs

```diff
@@ -86,6 +86,11 @@ private unsafe LazyStringValue CreateLazyStringValueFromParserState(JsonParserSt
             {
                 lazyStringValueFromParserState.EscapePositions = state.EscapePositions.ToArray();
             }
+            else
+            {
+                lazyStringValueFromParserState.EscapePositions = Array.Empty<int>();
```

Comment:
IIRC it can be set to `null`?

Reply:
It is assume that if the EscapePositions is null so the information about the escape characters placed in the bytes after the string bytes but if no such information were written this bytes are just garbage